components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/model/HttpRequestHeader.java

```diff
@@ -0,0 +1,42 @@
+package org.wso2.carbon.identity.oauth2.model;
+
+import java.io.Serializable;
+
+/**
+ * This class is used to store http request header information
+ */
+public class HttpRequestHeader implements Serializable {
```

Comment:
Serializable class should have a static final serialVersionUID field of type long. In IDEA there is a plugin to auto generate this.